File: Loenn/triggers/player_timestop_trigger.lua
local noMoveTrigger = {}

noMoveTrigger.name = "vitellary/nomovetrigger"

noMoveTrigger.fieldInformation = {
    stopLength = {
        minimumValue = 0.0
    }
}

noMoveTrigger.placements = {
    {
        name = "no_move_trigger",
        data = {
            stopLength = 2.0
        }
    }
}

return noMoveTrigger

File: Ahorn/triggers/playerTimestopTrigger.jl
module NoMoveTrigger

using ..Ahorn, Maple

@mapdef Trigger "vitellary/nomovetrigger" NoMove(x::Integer, y::Integer, width::Integer=Maple.defaultTriggerWidth, height::Integer=Maple.defaultTriggerHeight, stopLength::Number=2.0)

const placements = Ahorn.PlacementDict(
    "Player Timestop Trigger (Crystalline)" => Ahorn.EntityPlacement(
        NoMove,
        "rectangle"
    )
)

end